x/auth/ante/expected_keepers.go

```diff
@@ -30,3 +31,7 @@ type AccountKeeper interface {
 type FeegrantKeeper interface {
 	UseGrantedFees(ctx context.Context, granter, grantee sdk.AccAddress, fee sdk.Coins, msgs []sdk.Msg) error
 }
+
+type ConsensusKeeper interface {
```

Comment:
I can't find usages, it should be deleted?

Reply:
removed